apps/io-services-cms-webapp/src/webservice/index.ts

```diff
@@ -50,7 +50,7 @@ import {
 } from "./controllers/get-service-lifecycle-internal";
 import {
   applyRequestMiddelwares as applyGetPublicationStatusServiceRequestMiddelwares,
-  makeGetServiceHandler,
+  makeGetServicePublicationHandler as makeGetServicePublicationHandler,
```

Comment:
typo? just `makeGetServicePublicationHandler` should work!

Reply:
fixed with 9f3507f1b4273a5c044530eec8ace71cc04f5413